app/templates/contact/confirmation.html

```diff
@@ -30,6 +30,59 @@
 
 
 {% block content %}
+  <script nonce="{{ csp_nonce() }}">
```

Comment:
Can this go into a script module? Also a request for some more scattered comments please to help understand what's happening here